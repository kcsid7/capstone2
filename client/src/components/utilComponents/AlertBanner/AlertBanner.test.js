import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom'; 

import AlertBanner from './AlertBanner';


test("Renders the Alert Banner properly", () => {
    render(<BrowserRouter><AlertBanner /></BrowserRouter>);
})


test("Alert banner snapshot", () => {
    const { asFragment } = render(<BrowserRouter><AlertBanner /></BrowserRouter>);
    expect(asFragment()).toMatchSnapshot();
})