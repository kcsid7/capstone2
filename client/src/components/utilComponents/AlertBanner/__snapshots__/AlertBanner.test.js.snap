// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert banner snapshot 1`] = `<DocumentFragment />`;
